### DISCUSSION TITLE
Images doesn't appear when using minio externally(not a local dev setup)

### DISCUSSION THREAD

**Discussion by bernardosts** [outline/outline#3464] [Self Hosting]
Hi guys!

I'm having some trouble with the uploaded images when using minio externally. 
When I upload a image, after saving the changes, the image doesn't appear. And while posting I see that the UI does a GET request to an attachments.redirect that is returning an location that is somehow strange to me(look image below):

![image](https://user-images.githubusercontent.com/16948485/165379457-caffc556-37e2-49a1-8e9d-5cea8cb73bae.png)

Do you have any clue about how could I control the url returned in the Location Header?(I'm suposing that's were the GUI will try to find the image to render, maybe I'm wrong also... )

Thanks in advance

**Comment by Limezy** [comment]
I think you should at least share here relevant extracts of your .env file if you want to get some help 😃

**Reply by bernardosts** [reply]
@Limezy sorry for the late answer.

I hope this might help:

![2022-05-13_10-21](https://user-images.githubusercontent.com/16948485/168293249-b181add2-d22b-4f97-b45c-60b5f5656ede.png)
